CMakeLists.txt

```diff
@@ -254,6 +254,7 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     -Wno-undefined-func-template #TODO fix
     -Wno-float-equal #TODO fix
     -Wno-conversion #TODO fix
+    -Wno-missing-variable-declarations #TODO fix
```

Comment:
Is there a reason this was added back in?  It was recently fixed in https://github.com/apache/geode-native/pull/645 with it only being applied to the [old integration tests](https://github.com/apache/geode-native/blob/develop/cppcache/integration-test/CMakeLists.txt#L110) to save hassle on the old stuff

Reply:
Thanks for catching that. I, for some reason, reverted the commit you made by mistake and got squash with revision 2.
I've fixed it thought in revision 3. I am currently executing the pipeline for the latests changes.